.github/pull_request_template.md

```diff
@@ -10,7 +10,7 @@ Short rational why preCICE needs this change. If this is already described in an
 ## Author's checklist
 
 * [ ] I added a changelog file with this PR number in `docs/changelog/` if there are noteworthy changes.
-* [ ] I ran `tools/formatting/check-format` and everything is formatted correctly.
+* [ ] I ran `tools/formatting/check-format` and everything is formatted correctly. Run `tools/formatting/format-all` to reformat files automatically.
```

Comment:
```suggestion
* [ ] I ran `tools/formatting/format-all` to ensure everything is formatted correctly.
```

Reply:
I was not sure if one suggests this, without any risk of messing something up. I never had the reformatting fail for me, but who knows. I also prefer the shorter/simpler solution. :smile: